### PR TITLE
Bookstore: Add Styling

### PR DESCRIPTION
**Pull request by JEFFKAY16**
## Bookstore: Add Styling
In this update, I did the following: 

- I styled my Bookstore project using the design on [Zeplin](https://zeplin.io/).

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -6,21 +6,6 @@ env:
...
-      - uses: actions/setup-node@v1
-        with:
-          node-version: "12.x"
-      - name: Setup ESLint
-        run: |
-          npm install --save-dev eslint@7.x eslint-config-airbnb@18.x eslint-plugin-import@2.x eslint-plugin-jsx-a11y@6.x eslint-plugin-react@7.x eslint-plugin-react-hooks@4.x @babel/eslint-parser@7.x @babel/core@7.x  @babel/plugin-syntax-jsx@7.x @babel/preset-env@7.x  @babel/preset-react@7.x
-          [ -f .eslintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/react-redux/.eslintrc.json
-          [ -f .babelrc ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/react-redux/.babelrc
-      - name: ESLint Report
-        run: npx eslint .
```
Comment (andersonlebon) [inline .github/workflows/linters.yml]: - [ ] Why did you change the linters configuration Kindly make sure you have included the proper configuration(react-redux)

image

File: src/components/AddBook.js [hunk 2]
```diff
@@ -32,12 +32,14 @@ export default function AddBook() {
...
-      <input type="submit" value="ADD BOOK" />
+      <div className="add-form horizontal-line-70">
+        <h2 className="title-form">ADD NEW BOOK</h2>
+        <input className="input-one" type="text" placeholder="Add Book" onChange={(e) => setTitle(e.target.value)} />
+        <select name="cars" id="cars">
+          <option value="Under Construction">Categories</option>
+        </select>
+        <input className="inout-two" type="submit" value="ADD BOOK" />
+      </div>
     </form>
```
Comment (andersonlebon) [inline src/components/AddBook.js]: image

You can solve this linters by adding the `key` property on these stages :point_up: 

### before adding key attributes.
image

### example after adding them

image

File: src/components/App.css [hunk 3]
```diff
@@ -23,39 +36,183 @@ h1 {
...
+  padding-right: 40px;
+}
+
+.center-info {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 20px;
+}
+
```
Comment (andersonlebon) [inline src/components/App.css]: - [x] As the requirements say that we should follow the template design, Then I think it would be better if you center your progress-bar container. kindly check here the difference :point_down: 

Your design | The template
----|----
image | image

File: src/components/Book.js [hunk 4]
```diff
@@ -21,34 +24,43 @@ function Book() {
...
                 </div>
-              </div>
-              <div className="center-info">
-                <div className="progres-icon">Progress Icon</div>
-                <div className="progress-data">
-                  <h4>64%</h4>
-                  <span>Completed</span>
+                <div className="second-info">
+                  <div className="center-info">
+                    {/* <div className="progres-icon">Progress Icon</div> */}
```
Comment (andersonlebon) [inline src/components/Book.js]: - [x] kindly remove commented code to make your code more readable.

File: src/components/Book.js [hunk 5]
```diff
@@ -1,6 +1,9 @@
 import React from 'react';
 import './App.css';
 import { useSelector, useDispatch } from 'react-redux';
+// import { CircularProgressbar } from 'react-circular-progressbar';
```
Comment (andersonlebon) [inline src/components/Book.js]: - [x]  kindly remove commented code to make your code more readable.

File: src/components/AddBook.js [hunk 6]
```diff
@@ -13,8 +13,8 @@ export default function AddBook() {
     const newBook = {
       item_id: uuidv4(),
       title: bookTitle,
-      category: 'Random',
-      author: 'Jeff',
+      category: 'Business',
+      author: 'Lewis Hamilton',
```
Comment (andersonlebon) [inline src/components/AddBook.js:17]: ## [OPTIONAL]

- [x] I would really suggest you implement the categories selection feature and not take the default value `business`

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @JEFFKAY16 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflows/linters.yml (hunk 1 above)
Comment (JEFFKAY16) [reply .github/workflows/linters.yml]: @andersonlebon Hi,

I did this because i contacted someone from Microverse and they said they have updated the linters for redux and react, so it's fairly to pass - but if I run them locally they are working.

File: src/components/AddBook.js (hunk 2 above)
Comment (JEFFKAY16) [reply src/components/AddBook.js]: Done

File: .github/workflows/linters.yml (hunk 1 above)
Comment (andersonlebon) [reply .github/workflows/linters.yml]: hey @JEFFKAY16  kindly note that I run them locally but they were not passing
image

File: src/components/AddBook.js (hunk 6 above)
Comment (JEFFKAY16) [reply src/components/AddBook.js:17]: @andersonlebon 

Thanks. It is not a requirement, I also left it like that because there is no provision for a user to add a category, so what much be rendered?

File: src/components/App.css (hunk 3 above)
Comment (JEFFKAY16) [reply src/components/App.css]: I have

File: .github/workflows/linters.yml (hunk 1 above)
Comment (JEFFKAY16) [reply .github/workflows/linters.yml]: Hey @andersonlebon 

Like I mentioned, the linters have been updated and there is nothing I can on my side.

Comment (andersonlebon) [reply .github/workflows/linters.yml]: Hey @JEFFKAY16 try to use this [configuration](https://github.com/andersonlebon/BookStore/blob/development/.github/workflows/linters.yml)

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi @JEFFKAY16, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mke2111 — COMMENTED** [review]
Hello @JEFFKAY16  👋,

_This is an additional review as a result of the second opinion request._

I'm glad that everything has been resolved and approved in this PR. Congratulations 🍸 

> If you need further clarification about anything that is not clear, please tag me in this thread.

Keep rocking 🚀 and Enjoy your weekend.
